cmd/image-processing/main_test.go

```diff
@@ -193,177 +166,132 @@ var _ = Describe("Image Processing Resource", func() {
 
 		It("should fail in case --image does not exist", func() {
 			Expect(run(
-				"--image",
-				"docker.io/feqlQoDIHc/bcfHFHHXYF",
+				"--image", "docker.io/feqlQoDIHc/bcfHFHHXYF",
 			)).To(HaveOccurred())
 		})
 
 		It("should fail in case annotation is invalid", func() {
-			tag := pushImage("test1")
```

Comment:
I figured there is no need to have hardcoded numbered test strings for the image tag, so the temporary images just get a random tag each and we should be good, especially since everything clears up at the end of the test anyway.